lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,74 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
   margin: 0;
-  padding-top: 20px;
   position: relative; // to make a containing block for validation popovers
 }
 
-.nx-form--simple {
-  border-top: none;
-  padding: 0;
-}
+.nx-form-footer {
+  @include container-horizontal;
 
-// for a form that is the entirety of a modal
-.nx-form--modal-form {
-  padding-top: 0;
+  border-top: 1px solid #e9e9e9;
+  padding: $nx-spacing-xl $nx-spacing-l;
+  text-align: right;
 }
 
 .nx-form-group {
-  margin-bottom: 20px;
+  margin-bottom: 6px;
 }
 
 .nx-form-row {
   @include container-horizontal;
-  margin-bottom: 24px;
+  margin-bottom: 6px;
 
   .nx-form-group, .nx-fieldset {
```

Comment:
Isn't `.nx-fieldset` no longer supposed to be used within `.nx-form-row`?

Reply:
> Isn't `.nx-fieldset` no longer supposed to be used within `.nx-form-row`?

No, no it is not: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/c23bc1d427120b13235d50e6dea8e5544be14744